README.rst

```diff
@@ -134,6 +134,6 @@ A good place to start contributing is by looking through the issues
 
 .. |Project Name| replace:: PyTensor
 .. |Tests Status| image:: https://github.com/pymc-devs/pytensor/workflows/Tests/badge.svg
-  :target: https://github.com/pymc-devs/pytensor/actions?query=workflow%3ATests
+  :target: https://github.com/pymc-devs/pytensor/actions?query=workflow%3ATests+branch%3Amain
```

Comment:
This seems to include main from forks: https://github.com/pymc-devs/pytensor/actions?query=workflow%3ATests+branch%3Amain